src/components/candidateContent/candidateInfo/CandidateInfo.jsx

```diff
@@ -13,7 +13,7 @@ import { getFieldLabel } from '../../../utils';
 
 import { tableCandidateInfoFields } from '../../../constants/tableCandidateInfoFields';
 
-export const CandidateInfo = (props) => {
+const CandidateInfo = (props) => {
```

Comment:
why did you change the export to default?

Reply:
We export everywhere by default. So that everyone has the same. For consistency ...